src/gc/impl/conservative/gc.d

```diff
@@ -1752,7 +1758,7 @@ struct Gcx
                     minimize();
                 }
             }
-            else
+            else if (usedLargePages > 0)
```

Comment:
Same thing here, but it might be technically more likely to have this happen if you use up all your memory with small allocations. And only on 32-bit systems.

Really, I feel this extra check is not worth it. As this only is true potentially once.

Reply:
Same for the very first pool for large objects.